buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/DistributionTest.kt

```diff
@@ -63,6 +66,37 @@ open class DistributionTest : Test() {
         jvmArgumentProviders.add(libsRepository)
         systemProperty("java9Home", project.findProperty("java9Home") ?: System.getProperty("java9Home"))
     }
+
+    override fun executeTests() {
```

Comment:
❌ Note that `PerformanceTest`, `SmokeTest` and `SoakTest` are also `DistributionTest`s. I guess we should add the time to `IntegrationTest`s only, right?